src/lndUtils.ts

```diff
@@ -444,7 +443,8 @@ export const validate = async ({
     // TODO: if expired_at expired, thrown an error
   } else {
     if (!params.username) {
-      const error = `a username is required for push payment to the ${yamlConfig.name}`
+      const name = getName()
```

Comment:
Here too... may as well clean things up if we are touching this code. Something more descriptive like `galoyInstanceName` might be appropriate?

Reply:
Agreed :+1: